pyeudiw/oauth2/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class KidError(Exception):
```

Comment:
please move to jwk since the kid is defined in jwk and not in oauth2

Reply:
Resolved in ec43f2e